.gitignore

```diff
@@ -14,6 +14,7 @@ node_modules
 /.pnp
 .pnp.js
 .pnpm-debug.log
+.pnpm-store
```

Comment:
made a change to .gitignore because it seems like pnpm-store shouldn't be commited. Let me know if that's ok. I can remove it otherwise